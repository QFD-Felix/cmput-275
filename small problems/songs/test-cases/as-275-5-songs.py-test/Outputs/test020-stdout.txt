13 7 3 8 6 11 19 29 42 5 46 6 3 6 24 3 18 6 11 9 1 10 0 1 18 16 0 0 7 1 3 55 0 4 10 8 22 23 8 0 0 6 33 2 8 3 13 6 1 6 1 11 13 16 0 6 1 16 5 8 4 10 12 17 2 3 29 3 26 24 12 5 0 5 15 2 9 2 9 47 1 2 2 34 8 6 10 1 2 30 39 11 5 10 13 4 12 3 0 10 1 4 2 3 39 6 0 9 14 4 0 3 0 20 1 3 11 8 6 1 3 0 6 0 22 11 23 11 3 3 0 17 5 1 7 0 3 3 0 0 4 15 0 8 3 0 0 0 2 0 4 1 3 0 3 0 0 0 0
