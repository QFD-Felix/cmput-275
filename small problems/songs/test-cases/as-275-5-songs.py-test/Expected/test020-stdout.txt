13 7 3 8 6 11 19 29 42 5 46 6 3 6 24 3 18 6 11 9 1 10 0 1 18 16 0 0 7 1 3 57 0 4 10 8 22 23 8 0 0 6 35 2 8 3 13 6 1 6 1 11 11 18 0 6 1 18 7 10 6 12 14 19 0 2 2 3 31 3 28 26 12 5 0 5 17 2 9 2 11 50 1 2 2 37 3 3 12 3 6 4 2 31 40 11 5 10 13 4 12 3 0 10 1 4 2 3 40 6 0 10 15 5 0 4 0 0 20 1 3 11 8 6 1 3 0 6 0 22 11 23 11 3 3 0 17 5 1 7 0 3 3 0 0 4 15 0 8 3 0 0 0 2 0 4 1 3 0 3 0 0 0 0
